Sasquatch/SasquatchUITests/AppCenterUITests.swift

```diff
@@ -171,13 +171,13 @@ class AppCenterUITests: XCTestCase {
     app.alerts.element.buttons["OK"].tap()
   }
 
+  #warning("TODO: Uncomment when \"Device Info\" will be available. There is no UI now.")
```

Comment:
Can't `#warning` be in a func? Seems it is odd to have TODO comment specific to a line at the top. We should think about using `#warning` again.

Reply:
In a func it doesn't show XCode warning. May be roll back the addition of #warnings?

Reply:
Yeah.. Let's roll back. It will be super confusing if we look at the code later. I mean it will be hard to find.